bot.py

```diff
@@ -667,6 +669,8 @@ def check(m):
                                    "`{}role gunner` for more information.".format(
                             session[1][player][4].count('bullet'), '' if session[1][player][4].count('bullet') == 1 else 's',
                             BOT_PREFIX))
+                    if role == 'matchmaker' and sendrole:
```

Comment:
You can add matchmaker to the list of roles on line 664 and this will automatically be factored in